store/mockstore/mocktikv/executor_test.go

```diff
@@ -93,6 +93,12 @@ func (s *testExecutorSuite) TestResolvedLargeTxnLocks(c *C) {
 	// After that, the query should read the previous version data.
 	tk.MustQuery("select * from t").Check(testkit.Rows("1 1"))
 
+	// Cover BatchGet.
+	tk.MustQuery("select * from t where id in (1)").Check(testkit.Rows("1 1"))
+
+	// Cover PointGet.
+	tk.MustQuery("select * from t where id = 1").Check(testkit.Rows("1 1"))
```

Comment:
Does this query use MaxTS?
Then we need also to cover PointGet with real TS.

Reply:
Done